airflow/utils/log/logging_mixin.py

```diff
@@ -68,14 +68,43 @@ class LoggingMixin:
     """Convenience super-class to have a logger configured with the class name."""
 
     _log: logging.Logger | None = None
+    _parent_logger: str | None = None
```

Comment:
I feel this should be `_parent_logger_name` for consistency.

Reply:
Sat it to `_parent_logger_name`, thanks !